<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Spiel Starten</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic1_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic2_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>601</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Spielereingabe1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpieler1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>105</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spieler 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpieler2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>105</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spieler 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpieler3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>105</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spieler 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpieler4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>105</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spieler 4:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Spielereingabe2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="Spielereingabe3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="Spielereingabe4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="pic1_label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infolabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Bitte kreuze für die gewünschte Anzahl von Spielern eines der unten stehenden Kästchen an. Anschließend kannst du die Spielernamen vergeben (maximal 12 Zeichen).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2Spieler">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2 Spieler</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3Spieler">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3 Spieler</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4Spieler">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4 Spieler</string>
    </property>
   </widget>
   <zorder>pic1_label_2</zorder>
   <zorder>pic1_label</zorder>
   <zorder>button</zorder>
   <zorder>pic2_label</zorder>
   <zorder>Spielereingabe1</zorder>
   <zorder>labelSpieler1</zorder>
   <zorder>labelSpieler2</zorder>
   <zorder>labelSpieler3</zorder>
   <zorder>labelSpieler4</zorder>
   <zorder>Spielereingabe2</zorder>
   <zorder>Spielereingabe3</zorder>
   <zorder>Spielereingabe4</zorder>
   <zorder>infolabel</zorder>
   <zorder>radioButton_2Spieler</zorder>
   <zorder>radioButton_3Spieler</zorder>
   <zorder>radioButton_4Spieler</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMemory">
    <property name="title">
     <string>Memory</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionSpielregeln"/>
   </widget>
   <addaction name="menuMemory"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionSpielregeln">
   <property name="text">
    <string>Spielregeln</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
